### PR TITLE
search 7.x | custom facet improvements | LRDOCS-8675

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-8675

### CODE REVIEW

File: docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-facets/custom-facet.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Custom Facet
 
+7.3 FP1+ and the new GA for CE will fix DDM fields so we should document this, how to use DDM fields with the custom facet for Web Content
+7.2: By default it's legacy mode so DDM works, only if you switch to the newer nested field style (using the configuration) you will need to use the nested fields as well.
+7.2: enable Nested Fields Storage, now custom facet doesn't work FP8+ (unless the fix is backported)
+
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-facets/custom-facet.md:6]: :point_up: These are supposed to be notes? :thinking: They don't seem like text intended for readers.

File: docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-facets/custom-facet.md [hunk 2]
```diff
@@ -39,14 +43,22 @@ Advanced Configuration contains additional options:
...
+To use the Custom Facet, you must know which non-analyzed keyword field to use in the configuration. 
+
+```tip::
+   Elasticsearch supports indexing fields in multiple ways. Some text fields can be used as keyword fields if they're nested ``raw`` `multi-fields <https://www.elastic.co/guide/en/elasticsearch/reference/7.x/multi-fields.html>`__ in the mapping, or if the field is mapped in an additional separate field mapping as ``fieldName_sortable`` (as a ``keyword``). See the example below on creating facets for Custom Fields, as it leverages the Elasticsearch multi-field concept.
+```
+
+To browse the entire list of available fields, inspect the field mappings from *Control Panel* &rarr; *Configuration* &rarr; *Search* (click the *Field Mappings* tab). Here you'll see numerous indexes. The Liferay Assets you're likely interested in are indexed to the [company index](../../liferay-enterprise-search/cross-cluster-replication/cross-cluster-replication.md#liferay-dxp-decide-which-indexes-to-replicate-from-the-remote-cluster), which looks is named similarly to `liferay-20101` (`20101` is the Company ID).
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-facets/custom-facet.md:53]: Do not link to the CCR docs from here. We should create a dedicated article which deals with the index types instead.

File: docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-facets/custom-facet.md [hunk 3]
```diff
@@ -81,4 +93,99 @@ Here's a snippet of output from the Elasticsearch example:
...
+```bash 
+GET /liferay-20097/_mapping/field/*.raw
+```
+
+Setting a custom field to searchable means that the value of the field is indexed when the entity is modified, or when a re-index is triggered. Only `java.lang.String` fields can be made searchable.
+
+## Accessing Nested DDM Fields
+
+As documented in the [7.3 Breaking Changes document](../../../liferay-internals/reference/7-3-breaking-changes.md#dynamic-data-mapping-fields-in-elasticsearch-have-changed-to-a-nested-document), the way Liferay Dynamic Data Mapping framework indexes some fields has changed---where they were previously indeed at the root of the search engine document, they're now nested fields. This change affects Liferay 7.3 and Liferay 7.2 SP3/FP8+. On the latest Fix Pack and GA release of 7.3, this change is accounted for in Liferay's Search API and no configuration updates are necessary. Therefore, if you have Custom Facet widgets that relied on fields named `ddm__text__*` or `ddm__keyword__*` that were at the root of the Elasticsearch document, continue to use these fields as usual in your Custom Facet's _Aggregation Field_ configuration, even though they're no longer at the root of the document.
+
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-facets/custom-facet.md:147]: Add note that in 7.2 FP8+/SP3+ the Legacy Mode must be disabled to use the nested field structure.

**Comment by lipusz** [comment]
See my comments.

File: docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-facets/custom-facet.md (hunk 1 above)
Comment (rbohl) [reply docs/dxp/7.x/en/using-search/search-pages-and-widgets/search-facets/custom-facet.md:6]: yes these appear to be notes--it was too long ago so i don't recall what i'm talking about :sweat_smile: 

i'll look back at this

**Pull request closed without merging**